### PR TITLE
!faucet command (#445)

### LINKED ISSUE #445: add !faucet command to console
for testnet we should have a faucet command that shows a list with the following options

http://faucet.ma.cx:3000/
https://zerogox.com/ethereum/wei_faucet
http://icarus.parity.io/rain/0x<address>

`curl -X POST  -H "Content-Type: application/json" -d '{"toWhom":"0x<address>"}' https://morden.faucet.b9lab.com/tap

### PR DESCRIPTION
**Pull request by alwx**
Fixes #445

### CODE REVIEW

**Comment by alwx** [comment]
It's tricky, but it works.
We can prefill some fields on webpage or call API method (which was muuuuch easier to implement). 

There are 2 faucets available (because, unfortunately, some faucets from the original task description currently don't work), but it is easy enough to add more if we need.

**Comment by rasom** [comment]
There should be validation, no sense to allow sending of this command until account isn't created

**Comment by rasom** [comment]
It is possible to send `!faucet` command with arbitrary content
screen shot 2016-12-07 at 18 58 23

**Comment by alwx** [comment]
@rasom can now be checked

File: src/status_im/chat/handlers/faucet.cljs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  [faucet-name current-address {:keys [api-url]}]
+  (let [api-url (gstring/format api-url current-address)]
+    (http-get api-url
+              #(received-message (label :t/faucet-success))
+              #(received-message (label :t/faucet-error)))))
+
+(defmethod open-faucet :prefill
+  [faucet-name current-address {:keys [prefill-js]}]
+  (let [prefill-js  (gstring/format prefill-js current-address)
+        web-view-js (gstring/format "window.onload = function(){ %s };" prefill-js)]
```
Comment (rasom) [inline src/status_im/chat/handlers/faucet.cljs]: Why do we need to make this magic, is it possible just to send request using something like this https://github.com/status-im/status-react/blob/3122cb7ac3c63ac500d9a6147ce0afa9a623b8c5/src/status_im/utils/utils.cljs#L21-L21 ? Maybe, in this case, we will be able to get some useful info from response

Comment (alwx) [inline src/status_im/chat/handlers/faucet.cljs]: No, it is not possible, unfortunately. There is an extra form parameter and they check this paramer when you send the form.
screen shot 2016-12-08 at 13 50 55

For example, this query works:
```curl --data "ethereum_address=24c4d1b07da971c037d3db3f3837fd840037acd6&form_build_id=form-V-rIjQnyT1eErKXbnmJwW4lVr7SlQg6PenYPrDBQmC0&form_id=ethereum_wei_faucet_form" https://zerogox.com/ethereum/wei_faucet```

but this one doesn't work
```curl --data "ethereum_address=24c4d1b07da971c037d3db3f3837fd840037acd6&form_id=ethereum_wei_faucet_form" https://zerogox.com/ethereum/wei_faucet```

Comment (rasom) [inline src/status_im/chat/handlers/faucet.cljs]: got it, thanks

**Comment by rasom** [comment]
It seems that address input was not prefilled

z

Maybe it is better to use only `http://faucet.ropsten.be:3001`?

**Comment by rasom** [comment]
@jarradh what do you think?

**Pull request merged**